ui/src/app/settings/views/Dhcp/DhcpList/DhcpList.js

```diff
@@ -222,6 +224,16 @@ const DhcpList = ({ initialCount = 20 }) => {
     dispatch(machineActions.fetch());
   }, [dispatch]);
 
+  useEffect(() => {
+    if (saved && deletingName) {
+      dispatch(
+        messages.add(`${deletingName} removed successfully.`, "information")
```

Comment:
We should probably make the message types a const somewhere. No need to do that here though.

Reply:
Good idea, I've added: https://github.com/canonical-web-and-design/maas-ui/issues/181